### DISCUSSION TITLE
Boas-vindas ao Ivan

### DISCUSSION THREAD

**Discussion by pcosme** [pcosme/TETHYS-Graphene-Hydrodynamic-Simulation#68] [General]
@ifgrd26 agora que te temos como novo colega aqui a trabalhar no código vou deixar este _issue_ aberto para se quiseres deixares as tuas dúvidas, sugestões  &c e assim podemos criar um mini forum de discussão com o @jrebelosantos também. 

Espero que o código não esteja muito confuso :wink: prometo que vou tentar comentá-lo mais

**Comment by ifgrdo** [comment]
muito obrigado! :grinning:

**Comment by pcosme** [comment]
@jrebelosantos  e @ifgrd26  entretanto criei então um branch novo chamado TestingArea onde podemos fazer o que quisermos sem medo de estragar ;)

**Comment by ifgrdo** [comment]
assim já é mais seguro, não há o risco de estragar nada ;)

**Comment by pcosme** [comment]
Oi @ifgrd26 olha, entretanto em termos de análise dos resultados estou a voltar a usar Dynamic Mode Decomposition que é algo que não é de todo falado no curso por isso se quiseres ficar com uma ideia podes espreitar [aqui](http://www.aimsciences.org/article/doi/10.3934/jcd.2014.1.391) e claro na [wikipedia](https://en.wikipedia.org/wiki/Dynamic_mode_decomposition) 

Não me vou dar ao trabalho de tentar implementar em C++ (já passei por isso acredita não é bom :sweat_smile:) e portanto acho que vamos simplesmente usar o [pyDMD](https://mathlab.github.io/PyDMD/)

**Comment by ifgrdo** [comment]
Hey, obrigado pelos links.
Amanhã começo a ver isso
